bedrock/mozorg/templates/mozorg/moss/index.html

```diff
@@ -31,8 +31,8 @@
     <h2 class="moss-section-heading">2020 Funding Update</h2>
 
     <p>
-    {% with blogpost='https://blog.mozilla.org/blog/2020/08/11/changing-world-changing-mozilla/' %}
-      Due to the <a href="{{ blogpost }}">recently-announced restructuring at Mozilla</a>, MOSS is not currently accepting applications and will not be making any new awards this year. We are conducting a comprehensive review of the program and hope to share information about future award cycles in 2021.
+    {% with blogpost='https://blog.mozilla.org/blog/2020/08/11/changing-world-changing-mozilla/' mtf='https://foundation.mozilla.org/en/what-we-fund/awards/mozilla-technology-fund-mtf/' %}
```

Comment:
fixup: Comma needed when there are multiple arguments in a `with` clause. Page fails without this tweak

```suggestion
    {% with blogpost='https://blog.mozilla.org/blog/2020/08/11/changing-world-changing-mozilla/', mtf='https://foundation.mozilla.org/en/what-we-fund/awards/mozilla-technology-fund-mtf/' %}
```